docs/ref/clvm.md

```diff
@@ -633,31 +757,36 @@ A transaction block is considered "full" when it contains 2000 outputs. For this
 
   > Note: A transaction with only one input and one output is also possible. In theory, a block could therefore hold up to 2000 transactions, in which case the network would process an average of 38.5 (2000/51.95) transactions per second.
 
-With this goal in mind, Chia has created a generator program that processes 2000 compressed inputs and outputs. This program simulates a "full block".
+With this goal in mind, Chia has created a **generator program** that processes 2000 compressed inputs and outputs. This program simulates a "full block".
 
 To calculate the total amount of time for a Raspberry Pi 4 to process a full block, we must take into account three factors:
-* The time required to run the generator program (2000 inputs and outputs): 5.2 seconds
-* The time required to validate 2000 public keys: 2.2 seconds
-* The time required to validate 2000 aggregate signatures: 10.63 seconds
+* The time required to run the generator program (2000 inputs and outputs)
+  * The Raspberry Pi 4 accomplishes this in 5.2 seconds
+* The time required to validate 2000 public keys
+  * 2.2 seconds
+* The time required to validate 2000 aggregate signatures
+  * 10.63 seconds
 
-Therefore, the total amount of time required for a Raspbery Pi 4 to process a full block is 5.2 + 2.2 + 10.63 = 18.03 seconds. This is 10.095 seconds faster than the minimum time between blocks, and 33.92 seconds faster than the average. When considering other factors such as network latency and time required to fetch a full proof ([640 ms on a slow HDD](https://docs.chia.net/docs/03consensus/proof-of-space#farming)), this leaves plenty of leeway for a Raspberry Pi 4 to stay synced and collect farming rewards.
+Therefore, the total amount of time required for a Raspberry Pi 4 to process a full block is 5.2 + 2.2 + 10.63 = 18.03 seconds. This is 10.095 seconds faster than the minimum time between blocks, and 33.92 seconds faster than the average. When considering other factors such as network latency and time required to fetch a full proof ([640 ms on a slow HDD](https://docs.chia.net/docs/03consensus/proof-of-space#farming)), this still allows plenty of leeway for a Raspberry Pi 4 to stay synced and collect farming rewards.
 
-## Can a Raspberry Pi 4 sync from genesis?
+## Maximum cost per block rationale
 
-A Raspberry Pi 4 has four cores, so it can validate a pre-existing block in 18.03 / 4 = 4.5075 seconds, which is around 11.5 times the real-time rate of 51.95 seconds. Even in the worst-case scenario where every transaction block is full, the Pi can sync faster than the chain is being created.
+A Raspberry Pi 4 has four cores, so it can validate a pre-existing block in 18.03 / 4 = 4.5075 seconds, which is around 11.5 times the average real-time rate of 51.95 seconds. Even in the worst-case scenario where every transaction block is full, the Pi can sync faster than the chain is being created.
 
-## Calculating the maximum cost per block
+### Calculating the maximum cost per block
```

Comment:
this title, however, seems like a good candidate to be updated to "Maximum cost per block rationale".

Saying "calculating" here doesn't feel quite right. This section is really where we describe the rationale or how we determined the maximum block cost. Maybe "calculating" can be replaced by "determining" or "defining"